servicex/templates/did-finder-rucio/deployment.yaml

```diff
@@ -34,6 +34,10 @@ spec:
     {{- else }}
           - name: RMQ_URI
             value: amqp://user:{{ .Values.rabbitmq.auth.password }}@{{ .Release.Name }}-rabbitmq:5672
+    {{- end }}
+    {{- if .Values.didFinder.rucio.cachePrefix }}
+          - name: CACHE_PREFIX
```

Comment:
The Rucio DID Finder uses a [command line arg](https://github.com/ssl-hep/ServiceX_DID_Finder_Rucio/blob/519b3f52ce338870c8a3f34498947b5b592a1a74/scripts/did_finder.py#L46) for this, not an env var... this should go into the command

Reply:
The container calls a runme.sh that then calls the did finder script with the RMQ_URI command line.  This just follows that pattern and avoids having to parse args in a bash script.